packages/editor/src/components/post-publish-button/index.js

```diff
@@ -45,7 +90,14 @@ export class PostPublishButton extends Component {
 			onSubmit = noop,
 			onToggle,
 			visibility,
+			hasNonPostEntityChanges,
+			postType,
+			postId,
 		} = this.props;
+		const {
+			entitiesSavedStatesCallback,
+		} = this.state;
+
 		const isButtonDisabled =
```

Comment:
I think this needs to be accounting for `hasNonPostEntityChanges`.

Otherwise, if I have a post which includes _only_ changes to a non-post entity, it still appears as disabled:

![image](https://user-images.githubusercontent.com/1779930/70754412-2de13f80-1d05-11ea-954c-74d6d6d91b73.png)

But I can still click on it to show the save prompt:

![image](https://user-images.githubusercontent.com/1779930/70754447-3b96c500-1d05-11ea-964d-5472f38f4491.png)

Reply:
Fixed.